<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>785</width>
    <height>594</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>普通按钮</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>GroupBox</string>
        </property>
        <widget class="QCheckBox" name="btn996">
         <property name="geometry">
          <rect>
           <x>12</x>
           <y>113</y>
           <width>51</width>
           <height>19</height>
          </rect>
         </property>
         <property name="text">
          <string>996</string>
         </property>
        </widget>
        <widget class="QCheckBox" name="nomoney">
         <property name="geometry">
          <rect>
           <x>12</x>
           <y>225</y>
           <width>87</width>
           <height>19</height>
          </rect>
         </property>
         <property name="text">
          <string>无加班费</string>
         </property>
        </widget>
        <widget class="QCheckBox" name="nogirl">
         <property name="geometry">
          <rect>
           <x>12</x>
           <y>337</y>
           <width>87</width>
           <height>19</height>
          </rect>
         </property>
         <property name="text">
          <string>没有女生</string>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="checkBtn">
        <property name="text">
         <string>check 按钮</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QToolButton" name="toolButton">
        <property name="text">
         <string>tool按钮</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="toolButton_check">
        <property name="text">
         <string>tool check 按钮</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="toolButton_menu">
        <property name="text">
         <string>下拉 tool按钮</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="toolButton_action">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="toolButton_arrow">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QCheckBox" name="checkBox_all">
         <property name="text">
          <string>all</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_2" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="leftMargin">
           <number>40</number>
          </property>
          <item>
           <widget class="QCheckBox" name="checkBox">
            <property name="text">
             <string>1</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkBox_2">
            <property name="text">
             <string>2</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkBox_3">
            <property name="text">
             <string>3</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>785</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
